firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerformance.java

```diff
@@ -191,6 +193,14 @@ public static FirebasePerformance getInstance() {
     gaugeManager.setApplicationContext(appContext);
 
     mPerformanceCollectionForceEnabledState = configResolver.getIsPerformanceCollectionEnabled();
+    if (isPerformanceCollectionEnabled()) {
+      Log.i(
+          LogWrapper.LOG_TAG,
+          String.format(
+              "Welcome to Firebase Performance Monitoring! Please visit %s in a minute to see details.",
```

Comment:
This definitely needs to be changed. As a default, below is my recommendation.

"Firebase performance successfully initialized. Visit %s in a minute to know the performance metrics of this application run".

Reply:
Sounds good! I have asked Rachel to take a look at the sentence.

Reply:
Updated the sentence.